### DISCUSSION TITLE
Reverse X VS Y roll display in Step Failure

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#564] [Ideas]
When the Step Failure screen shows you what you rolled vs. what the equipment's Safety factor is, it shows the Safety first, then the result of your roll.  I can see how that would seem sensible from the programmer's point of view, but when I think of this sort of thing I think "I rolled Y and the component Safety is X", but the screen shows it the other way around.  In the screenshot below the Vostok, of course, had a 78% Safety.  Am i right in thinking it might be better to reverse the order - so this would have shown "93 VS 78"?

![X VS Y](https://user-images.githubusercontent.com/3118014/129300548-a9b2cdac-0b82-4a0d-b35e-8720abc84a4e.png)

**Comment by peyre** [comment]
Perhaps even better would be "93 VS. 78%" to highlight that 78 is the current Safety.

**Comment by hweimer** [comment]
Actually, 78% is 0.78, so leaving out the percent sign would be better. Changing the order is fine, though.

**Comment by peyre** [comment]
Uh oh!  I implemented this yesterday.  Do you think I should take out the percent sign?  Sure 78%=.78, but most players will be thinking in percents rather than decimals.  I think the percent sign helps cue the player that that's the Safety figure.

![Y VS X](https://user-images.githubusercontent.com/3118014/130366762-f5ef646c-c0fe-4e77-9b95-e5126ba69acb.png)

**Comment by hweimer** [comment]
I think it would be better without the percent sign. It's also not the safety factor that people will be familiar with from the R&D building or the mission announcement, since the number shown in the failure screen includes modifications from crew skill, penalties, etc.

**Reply by peyre** [reply]
Oh that's a good point; I suppose that figure would include those modifications.  I'll have to take it out this weekend.

**Reply by peyre** [reply]
I took out the percent sign on Saturday.
